Week-2/Exercise2/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="./style.css">
+</head>
+
+<body>
+    <div id="root">
+        <div id="parent">
+            <div id="content">Hi!</div>
+            <div id="red-square"></div>
+            <div id="yellow-square"></div>
+            <div id="blue-square"></div>
+            <div id="green-square"></div>
```

Comment:
why is there a need of id's here , when you can give classnames

Reply:
Need is to give styles only for that element.